CHANGES.rst

```diff
@@ -47,6 +47,8 @@ astropy.io.ascii
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Fix handling of 1-dimensional arrays with single elements in BinaryTableHDUs [#10768]
```

Comment:
```suggestion
- Fix handling of 1-dimensional arrays with single elements in ``BinTableHDU``s [#10768]
```
Suggest to use the class name; otherwise looks good once it's moved to the new milestone.

Reply:
Both done